### PR TITLE
Display links

### LINKED ISSUE #6: Link to contract and directions on how to mint
Users want to know how to take action with the given information.

- [ ] link to contract if available to mint
- [ ] include directions on how to mint - probably a rich text field we can edit via admin tool
- [ ] link to open sea collection if possible

### PR DESCRIPTION
**Pull request by str1nker**
Close #6 - Display links
Add post deploy script and JSON file with Projects data

### CODE REVIEW

File: backend/lib/data/projects_data.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "title": "Loot",
+    "nft_collection": {
+      "collection_url": "https://opensea.io/collection/lootproject"
+    },
+    "smart_contract": {
+      "contract_url": "https://etherscan.io/address/0xff9c1b15b16263c61d017ee9f65c50e4ae0113d7",
```
Comment (0xmunger) [inline backend/lib/data/projects_data.json:8]: this feels duplicative. is simply `contract_address` fine? with that we can derive the etherscan address

File: backend/lib/data/projects_data.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "address": "0xFF9C1b15B16263C61d017ee9F65C50e4AE0113D7"
+    },
+    "referral_projects": [
```
Comment (0xmunger) [inline backend/lib/data/projects_data.json:11]: a better word might be `derivative projects`

File: backend/lib/data/projects_data.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      {
+        "title": "Realms (for Adventurers)",
+        "nft_collection": {
```
Comment (0xmunger) [inline backend/lib/data/projects_data.json:14]: perhaps rename to `opensea_link`

**Review by 0xmunger — COMMENTED** [review]
thoughts on the schema being 

```json
{
   "name": "str",
   "description": "str",
   "contract_address": "str",
   "opensea_url": "str"
}

we could maybe add a bool for `is_erc20`? i think 99% of projects will _not_ be ERC-20


```

File: backend/lib/data/projects_data.json (hunk 3 above)
Comment (str1nker) [reply backend/lib/data/projects_data.json:14]: what if url is not OPENSEA? I believe we'll have more projects outside of that platform.  That's why at the beginning I added `Collection link` unstead of `Open Sea link` at the frontend. What if we want to redirect to `Cyberpunks` website for example?

File: backend/lib/data/projects_data.json (hunk 1 above)
Comment (str1nker) [reply backend/lib/data/projects_data.json:8]: yes, but what if we'll need to add more fields? This is broken down into associations. Maybe let's simplify like
`contract: { url: 'bla', address: 'bla' }`

File: backend/lib/data/projects_data.json (hunk 3 above)
Comment (0xmunger) [reply backend/lib/data/projects_data.json:14]: ah that makes sense. i think you're right collection_url is better...

**Pull request merged**